Misc/NEWS.d/next/Documentation/2024-03-03-17-04-30.gh-issue-115398.FQICYA.rst

```diff
@@ -0,0 +1,2 @@
+Suggest use of :func:`hasattr` when checking for availability of Expat reparse
```

Comment:
feel free to remove this file from the PR, we don't need a news entry for a doc update.

Reply:
@gpshead done